src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -235,7 +235,7 @@ const useGroupInfo = (channelId = null) => {
 
     try {
       const {channelName} = getChannelListInfo(
-        responseChannelData.data,
+        responseChannelData.data?.channel,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `responseChannelData.data` to `responseChannelData.data?.channel` is a good one as it adds an extra layer of null safety. This will prevent potential crashes if `data` is undefined.
```diff
- responseChannelData.data,
+ responseChannelData.data?.channel,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->